lib/android-helpers.js

```diff
@@ -467,11 +467,11 @@ helpers.initDevice = async function (adb, opts) {
   }
   if (!opts.avd) {
```

Comment:
> no, helpers don't have access to driver context

we could pass in the result of `isEmulator()` to this function as a param though...